proto/src/main/proto/grpc.proto

```diff
@@ -340,6 +342,14 @@ message GetUnusedBsqAddressReply {
      string address = 1;
 }
 
+message SendBsqRequest {
+    string address = 1;
+    double amount = 2;
```

Comment:
This doesn't make sense to me. BSQ amounts are always going to be an integer amount of satoshis and all other coin amounts in the pb spec are `uint64`.

Reply:
After much discussion on keybase, changed amount type to String in commit 65df9e1503a9a38518ac29544e8a62cd4ff1045a.

A user needs to be able to enter a representation of a decimal value like "0.25" instead of a satoshi value like 25000000.
The server will convert the string to a Coin for use in a transaction.